src/comps/Deposit.tsx

```diff
@@ -412,7 +408,12 @@ const DepositFlowConfirm = ({
           <SecondaryButton onClick={() => handlePrevClick()}>
             PREV
           </SecondaryButton>
-          <PrimaryButton onClick={() => handleNextClick()}>NEXT</PrimaryButton>
+          <PrimaryButton
+            disabled={isPendingNotifyEmily}
```

Comment:
Should we also disable `PREV` if there's a deposit in flight we are tracking?

Reply:
makes sense

Reply:
done